supervisor/shared/usb/tusb_config.h

```diff
@@ -89,6 +89,12 @@ extern "C" {
 #endif
 #endif
 
+// Use DMA with the USB peripheral.
+#ifdef CONFIG_IDF_TARGET_ESP32P4
+#define CFG_TUD_DWC2_DMA_ENABLE (1)
+#define CFG_TUH_DWC2_DMA_ENABLE (1)
```

Comment:
DMA can be enabled for S2/S3 as well as they share pretty much same usbip. 
- For P4, it use DCache and will need memory align 64 (variable must also occupies the whole cache line 64). 
- For S2/S3, aligne 4 bytes are good enough.